scripts/generate

```diff
@@ -590,19 +593,12 @@ ${Html.unwrap (toc (content))}
 ${content}
   </div>
   <script src="vendor/ramda.js"></script>
-  <script src="vendor/sanctuary-show.js"></script>
-  <script src="vendor/sanctuary-type-identifiers.js"></script>
-  <script src="vendor/sanctuary-type-classes.js"></script>
+  <script src="vendor/sanctuary-bundle.js"></script>
   <script src="vendor/sanctuary-descending.js"></script>
-  <script src="vendor/sanctuary-either.js"></script>
   <script src="vendor/sanctuary-identity.js"></script>
-  <script src="vendor/sanctuary-maybe.js"></script>
-  <script src="vendor/sanctuary-pair.js"></script>
-  <script src="vendor/sanctuary-def.js"></script>
   <script src="adt/Html.js"></script>
   <script src="adt/List.js"></script>
   <script src="adt/Sum.js"></script>
-  <script src="vendor/sanctuary.js"></script>
   <script src="env.js"></script>
```

Comment:
Now that we have __sanctuary-bundle.js__ we no longer need to enumerate Sanctuary's dependencies here.